seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3991,7 +3991,7 @@
       "tests": [
         "collection = collectionCopy; assert(update(5439, \"artist\", \"ABBA\")[5439][\"artist\"] === \"ABBA\", 'message: After <code>update(5439, \"artist\", \"ABBA\")</code>, <code>artist</code> should be <code>\"ABBA\"</code>');",
```

Comment:
This code is unchanged in this PR. This is currently deployed in the live version of the website and seems to be working fine. Does this test fail for you? Have you tested it out? Thanks.